v3/docs/multi-tenancy/delete-app.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Deleting an app
+hide_title: true
+sidebar_position: 5
+---
+
+import { BackendTabs } from "/src/components/Tabs";
+
+# Delete an app from SuperTokens core
+
+You can delete an app from your SuperTokens core. This operation is irreversible and will delete all user data associated with the app.
```

Comment:
Replace the first sentence with: "The following guide will show you how to delete an app from a SuperTokens Core instance". It's structured to behave like a mini introductory description for the guide